python_scripts/entries_updater.py

```diff
@@ -12,25 +12,38 @@
 from pymongo.errors import BulkWriteError
 
 uri = os.getenv("MONGODB_URI")
+# uri = "mongodb://user:2uHApc4xCLXTA8h@ds157723.mlab.com:57723/heroku_4rpt6s2b" #?retryWrites=false"
 # uri = "mongodb://localhost:27017/test" # Use this to test on local database.
 client = MongoClient(uri)
 database_name = pymongo.uri_parser.parse_uri(uri)['database']
 db = client[database_name]
 
-with open(os.path.join(os.path.dirname(__file__), 'consolidated_entries.json')) as f:
-    updates = json.load(f)
+entries = db.entries.find({})
```

Comment:
Can you uncomment out whatever code is needed so that running `python entries_updater.py` will update the appropriate entries and add the appropriate fields?